data/World/Shadow Temple.json

```diff
@@ -116,10 +138,28 @@
         "region_name": "Shadow Temple Before Boss",
         "dungeon": "Shadow Temple",
         "locations": {
+            "Shadow Temple After Boat Upper Recovery Heart 1" : "can_use(Distant_Scarecrow)",
+            "Shadow Temple After Boat Upper Recovery Heart 2" : "can_use(Distant_Scarecrow) ",
+            "Shadow Temple After Boat Lower Recovery Heart" : "can_use(Distant_Scarecrow) or ((Bow or (logic_shadow_statue and has_bombchus) or shadow_temple_shortcuts) and can_play(Song_of_Time))",
```

Comment:
You can get up to these with Hookshot if you go across the gap, then use Song of Time. The positioning is a bit precise but not too crazy. I'd say -probably- safe for default, but you might want to check if you agree.

Reply:
Yeah I tested this and it's definitely easy enough w/ hookshot. Didn't feel particularly precise

Reply:
Yes I agree. It's slightly precise in that you can just stand somewhere close and go for it, you actually have to try and get close, a little.